### DISCUSSION TITLE
部分网页仍然有部分页面未被正确重新渲染，请问如何解决

### DISCUSSION THREAD

**Discussion by KyleKwan2024** [F9y4ng/GreasyFork-Scripts#358] [问答专区(Question / Answer)] [answered]
![image](https://github.com/user-attachments/assets/5a4d1ef9-bf72-4096-a999-19d8a80165f9)
![image](https://github.com/user-attachments/assets/0ce52d29-bd3f-44db-9554-a76b2c599606)

已经用 “字体渲染（自用脚本）” 选择了腾讯体，并打开了字体重写。目标网页（内部OA系统）仍然有部分页面未被重新渲染为腾讯体。 比如：这个样式 <span class="mini-button-text  mini-button-icon icon-download" style="">批量下载 PDF文件</span>  请问是什么原因，以及如何解决？谢谢。

奇怪的是，脚本自带的霞鹜文楷却可以全部替换成功的。

Chrome 已是最新版本 126.0.6478.127（正式版本） ，脚本也是最新版本version 2024.07.06.1。

**Comment by F9y4ng** [comment, marked as answer]
我想你应该是没有完整的添加字体的postscript信息，所以才会出现字体热替换不完全。

建议你按 #261 中的方法获取字体的完整参数，然后重新添加字体数据。

**Reply by KyleKwan2024** [reply]
【重置脚本后再替换已成功，感谢！】
~~我是这样写的：{"ch":"腾讯体","en":"TencentSans","ps":"TencentSansW7"}
但结果是一样的。~~
![IMG_4807](https://github.com/user-attachments/assets/26ddd684-310d-40ed-b32c-98aea3fd6de4)
